src/components/table.ts

```diff
@@ -81,6 +81,20 @@ export module Components {
     }
 
     /**
+     * Returns the Component at the specified row and column index.
+     *
+     * @param {number} rowIndex
+     * @param {number} columnIndex
+     * @returns {Component} The Component at the specified position, or null if no Component is there.
+     */
+    public componentAt(rowIndex: number, columnIndex: number) {
+      if (rowIndex < 0 || rowIndex >= this._nRows || columnIndex < 0 || columnIndex >= this._nCols) {
+        return null;
+      }
+      return this._rows[rowIndex][columnIndex];
+    };
```

Comment:
Just curious (no code action needed, necessarily): let's say that we want to add an endpoint to set the component at a specified index, would we add it through extending this endpoint?